lib/samson/logging.rb

```diff
@@ -19,14 +19,14 @@
     {
       params: params,
       user_id: request.env['warden']&.user&.id,
-      ip: request.remote_ip
+      client_ip: request.remote_ip
```

Comment:
why did this change, just readability or something else ?

Reply:
I would guess readability. I mean, I wouldn't know looking at the logs if it was the server's IP (so you can track down which server is bad) or the clients IP (who causing the problem).